README.md

```diff
@@ -39,69 +43,131 @@ Pre-built docker for popular AI frameworks. Easy to include heterogeneous hardwa
+* OpenPAI reserves memory and CPU for service running, so make sure there are enough resource to run machine learning jobs. Check [hardware requirements](https://github.com/Microsoft/pai/wiki/Resource-Requirement) for details.
+* Dedicated servers for OpenPAI. OpenPAI manages all CPU, memory and GPU resources of servers. If there is any other workload, it may cause unknown problem due to insufficient resource.
+
+### Deploy
+
+The default deployment includes minimum steps to deploy an OpenPAI cluster. Base on the default configuration, you can customize OpenPAI deployment for different hardware environments and use scenarios.
+
+#### Deploy with default configurations
+
+If you are deploying a small or medium size cluster, which is less than 20 servers, it's recommended.
+
+If your are deploying a large size cluster, this section is still needed to generate default configuration, then customize on it.
+
+[Deploy with default configurations](docs/pai-management/doc/distributed-deploy.md)
+
+#### Customize deployment
+
+As various hardware environments and different use sencarios, default configuration of OpenPAI may not fit your case. Based on genereted default configration in last section, you can change it before deploying.
+
+[Customize deployment](docs/pai-management/doc/how-to-generate-cluster-config.md#Optional-Step-3.-Customize-configure-OpenPAI)
+
+### Validate deployment
+
+After deployed, it needs to check if key components in health status.
+
+[Validate deployment](docs/pai-management/doc/validate-deployment.md)
+
+After validated the deployment, you can submit a job to have a try, and check if it works end-to-end.
 
-- Ubuntu 16.04
-- Assign each server a static IP address. Network is reachable between servers.
-- Server can access the external network, especially need to have access to a Docker registry service (e.g., Docker hub) to pull the Docker images for the services to be deployed.
-- All machines' SSH service is enabled, share the same username / password and have sudo privilege.
-- Need to enable NTP service.
-- Recommend no Docker installed or a Docker with api version >= 1.26.
-- See [hardware resource requirements](https://github.com/Microsoft/pai/wiki/Resource-Requirement).
+[Submit a test job](examples/README.md#quickstart)
 
-#### 2 Deploy OpenPAI
+### FAQ
 
-If you have a cluster which contains more than 2 machine and want to deploy pai on it. Please choose ```Distributed deploy``` following.
+If there is any question durding deployment, check [here](docs/faq.md#deploy-and-maintenance-related-faqs) firstly.
 
-If you only have one mahince, and want to deploy pai on it. Please choose ```Single Box deploy``` following. 
+If FAQ doesn't resolve it, refer to [here](#get-involved) to ask question or submit an issue.
 
+## Train model
 
-##### 2.1 [Distributed deploy](./docs/pai-management/doc/distributed-deploy.md)
-##### 2.2 [Single Box deploy](./docs/pai-management/doc/single-box.md)
+### Why to schedule jobs
```

Comment:
This part should be move to "Why choose OpenPAI", only leave "How" here.

Reply:
"Why choose OpenPAI" is for decision maker. this part is to convince user, why schedule job on shared cluster is better than dedicated server. But it's a good idea to add corresponding content to Why part for decision maker also.